source/includes/administration/_quotas.md

```diff
@@ -0,0 +1,212 @@
+        {
+          "ceiling": 1.0,
+          "deleted": false,
+          "limitSize": 0,
+          "metricIdentifier": "metric-id",
+          "id": "1c005444-6ea2-4ebc-85f7-d75f5b1943d3",
+          "type": "INSTANCE_MEMORY",
+          "labelKey": "quotas.memory",
+          "version": 1
+        },
+        {
+          "ceiling": 1.0,
+          "deleted": false,
+          "limitSize": 0,
+          "metricIdentifier": "metric-id",
+          "id": "674f4755-640b-4697-87bb-8a5d03704552",
+          "type": "DISK_SIZE",
+          "labelKey": "quotas.disk_size",
+          "version": 1
+        },
+        {
+          "ceiling": 1.0,
+          "deleted": false,
+          "limitSize": 0,
+          "metricIdentifier": "metric-id",
+          "id": "691c55b5-ecaf-434d-b509-97c4fb554d1d",
+          "type": "INSTANCE_VCPUS",
+          "labelKey": "quotas.vcpus",
+          "version": 1
+        }
+      ],
+      "name": "Quota Name",
+      "defaultForService": true,
+      "description": "Quota Description",
+      "numberOfOrganizationsAppliedTo": 1,
+      "id": "24330f70-9845-4851-acc0-ecd5d9b13613",
+      "creationDate": "2021-01-06T19:17:06.000Z",
+      "version": 1,
+      "serviceConnection": {
+        "serviceCode": "service-code",
+        "name": "service-name",
+        "id": "52fb6687-44cb-4db3-9e63-bdfabfe2faf1",
+        "type": "gcp"
+      },
+      "defaultForTrial": true
+    }
+}
+```
+A quota with the following attributes are returned.
+
+Quota Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the quota.
+`name`<br/>*string* | The name of the quota.
+`description`<br/>*string* | The description of the quota.
+`creationDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) that the quota was created.
+`numberOfOrganizationsAppliedTo`<br/>*number* | The number of organizations the quota is assigned to.
+`defaultForService`<br/>*boolean* | Weather or not the quota is the default for service connections.
+`defaultForTrial`<br/>*boolean* | Weather or not the quota is the default for trials.
+`ownerOrganization`<br/>*[Organization](#administration-organizations)* | The organization that manages the quota.<br/>*includes*: `id` and `name`.
+`serviceConnection`<br/>*[ServiceConnection](#administration-service-connections)* | The service connection associated to the quota.<br/>*includes*: `id` and `name`.
+`quotaDetails`<br/>*Array [QuotaDetail]* | The quota details.
+
+The following table lists the attributes for `QuotaDetail`:
+
+QuotaDetail Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the quota detail.
+`type`<br/>*string* | The type of quota (metric).
+`labelKey`<br/>*string* | The key used for the label.
+`metricIdentifier`<br/>*string* | Id matching the metric identifiers provided by the plugin writer.
+`ceiling`<br/>*number* | The ceiling value (metric).
+`ceiling`<br/>*number* | The maximum for the type (metric).
```

Comment:
?

Reply:
I have a hard time with mac keybinding... duplicated rows. Sorry about that.

Reply:
Fixed in 1730927b87a08a48d2fa4a1ec102a15b5a0ee328